### DISCUSSION TITLE
rabbitmq_tracing: trace consumer acknowledgements?

### DISCUSSION THREAD

**Discussion by shihuncl** [rabbitmq/rabbitmq-server#12879] [Ideas] [answered]
### Is your feature request related to a problem? Please describe.

The rabbitmq plug-in traceing can only record published and received, but cannot record the ack consumed by the consumer?

### Describe the solution you'd like

record the ack consumed by the consumer

### Additional context

{"channel":1,"connection":"*****","exchange":"MQ_test_apisix_test_dev","node":"lcrabbitmq@8ba899f064c9","payload":"eyJtZXNzYWdlIjoidGVzdDU1NTU1In0=","properties":{},"queue":"none","routed_queues":["apisix_test"],"routing_keys":["apisix_test"],"timestamp":"2024-12-02T13:04:49.342+00:00","type":"published","user":"guest","vhost":"gt-dev"}
{"channel":1,"connection":"<lcrabbitmq@8ba899f064c9.1730191121.257.296>","exchange":"MQ_test_apisix_test_dev","node":"lcrabbitmq@8ba899f064c9","payload":"eyJtZXNzYWdlIjoidGVzdDU1NTU1In0=","properties":{},"queue":"apisix_test","routed_queues":"none","routing_keys":["apisix_test"],"timestamp":"2024-12-02T13:05:06.530+00:00","type":"received","user":"guest","vhost":"gt-dev"}

**Comment by michaelklishin** [comment, marked as answer]
@shihuncl simply [use Wireshark](https://www.rabbitmq.com/amqp-wireshark). I don't see a single reason to reinvent it. 

`rabbitmq_tracing` arguably should be removed from the distribution because it was never meant to be used for anything beyond development environments, and it was originally developed before Wireshark had the AMQP 0-9-1 dissector (protocol support). Wireshark can help you inspect AMQP 1.0, MQTT, STOMP and HTTP API requests as well.
